examples/multi_index_large/src/multi_index_large.cpp

```diff
@@ -0,0 +1,91 @@
+#include <multi_index_large.hpp>
+
+[[eosio::action]] 
+void multi_index_large::set( uint64_t id, uint64_t u64, uint128_t u128,
+         double f64, long double f128, eosio::checksum256 chk256 ) {
+   auto itr = testtab.find(id);
+   if ( itr == testtab.end() ) {
+      testtab.emplace( _self, [&]( auto& u ) {
+            u.id = id;
+            u.u64 = u64;
+            u.u128 = u128;
+            u.f128 = f128;
+            u.chk256 = chk256;
+      });
+   }
+}
+
+[[eosio::action]] 
+void multi_index_large::print( uint64_t id ) {
```

Comment:
why not pass in iter instead of id? The find can be saved.

Reply:
It is action executed from cleos